src/sentry/api/serializers/models/group.py

```diff
@@ -226,42 +235,58 @@ def get_attrs(self, item_list, user):
 
         seen_stats = self._get_seen_stats(item_list, user)
 
+        annotations_by_group_id = defaultdict(list)
+
+        # find every org that's part of the item list
+        organization_id_list = list(set(item.project.organization_id for item in item_list))
+
+        # find all the integration installs for those orgs that are have issue tracking
+        for integration in Integration.objects.filter(organizations__in=organization_id_list):
+            if not (
+                integration.has_feature(IntegrationFeatures.ISSUE_BASIC)
+                or integration.has_feature(IntegrationFeatures.ISSUE_SYNC)
+            ):
+                continue
+
+            for organization in integration.organizations.all():
```

Comment:
@markstory Were you talking about the loop here? I actually think this could be a problem because I could load the install for an unrelated org if they happen to have the same integration. Hmm....